pkg/controllers/targetversion/target_version_controller.go

```diff
@@ -27,6 +17,13 @@ import (
 	"github.com/openshift/library-go/pkg/operator/events"
 	"github.com/openshift/library-go/pkg/operator/status"
 	"github.com/openshift/library-go/pkg/operator/v1helpers"
+	appsv1 "k8s.io/api/apps/v1"
```

Comment:
order!

Reply:
now I know why @deads2k hate this comments :-)